src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflow.cs

```diff
@@ -6,16 +6,13 @@
 using Microsoft.Common.Core.Shell;
 using Microsoft.R.Components.Extensions;
 using Microsoft.R.Components.History;
-using Microsoft.R.Components.InteractiveWorkflow;
-using Microsoft.R.Components.InteractiveWorkflow.Implementation;
 using Microsoft.R.Components.PackageManager;
 using Microsoft.R.Components.Plots;
 using Microsoft.R.Components.Settings;
 using Microsoft.R.Host.Client;
-using Microsoft.R.Host.Client.Install;
-using Microsoft.VisualStudio.InteractiveWindow;
+using Microsoft.VisualStudio.R.Package.Repl;
 
-namespace Microsoft.VisualStudio.R.Package.Repl {
+namespace Microsoft.R.Components.InteractiveWorkflow.Implementation {
```

Comment:
Actually, I'd rather avoid Implementation and Definitions suffixes in namespaces. We need to clean those up.